addons/l10n_ch/tests/test_partner_bank_isr.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import time
+
+from odoo import tools
+from odoo.exceptions import ValidationError
+from odoo.modules.module import get_resource_path
+from odoo.tests import Form, common
+
+
+CH_ISR_SUBSCRIPTION = "01-162-8"  # partner ISR subsr num we register under postal
+CH_POSTAL = "10-8060-7"
+CH_IBAN = "CH15 3881 5158 3845 3843 7"
+
+
+class TestPartnerBankISR(common.SavepointCase):
+    """Test creation of partner bank account with l10n_ch fields"""
+
+    def setUp(self):
```

Comment:
I'd do this in `setUpClass` and as you create tracked records I'd disable tracking.

Reply:
Done